core/include/utility/Exception.hpp

```diff
@@ -16,9 +16,14 @@ namespace Utility
 		Unknown_Exception          = Exception_Unknown_Exception,
         Non_existing_Image         = Exception_Non_existing_Image,
         Non_existing_Chain         = Exception_Non_existing_Chain
+        // TODO: from Chain.cpp
+        // Last image deletion ?
+        // Empty clipboard     ?
 	};
   
-  void Handle_exception( const Exception & ex, int idx_image, int idx_chain );
+  void Spirit_Exception( const Exception & ex, int idx_image=-1, int idx_chain=-1 );
+
+  void Handle_Exception( int idx_image, int idx_chain );
```

Comment:
Most common case is not to have the `idx_image` in the call of the function in which there is a `catch`. Probably it would be better to re-arrange the function's arguments so we could use the c++ default arguments mechanism.
```c++
void Handle_Exception( int idx_chain=-1, int idx_image=-1 );
```
thus, we could avoid the initialization of a variable outside `int idx_image=-1` outside of the `try{ ... } catch{ ... }`